torch/fx/experimental/symbolic_shapes.py

```diff
@@ -3288,22 +3307,28 @@ def hint(s):
                 track_symint(source, t)
                 continue
             assert isinstance(t, Tensorlike)
+            sources_and_tensors = [(source, t)]
```

Comment:
Double checking my understanding: this used to be set up so that if `t` was a wrapper subclasses, `sources_and_tensors` would only be populated by the inner (plain tensors). Vs. now, `sources_and_tensors` will contain both the `JaggedTensor` subclass, **and** its inner values and offsets tensors (and corresponding source). We need this now because we also need to potentially generate guards on the outer JaggedTensor's ragged dimension. Is that right?

Reply:
Exactly. Updating the comment to clarify that a bit.